src/utils/fs/file-info.js

```diff
@@ -12,7 +11,7 @@ import type { PathOsBased } from '../path';
  *  currentDirName() // => 'bit'
  * ```
  */
-export default function calculateFileInfo(relativePath: string): PathOsBased {
+export default function calculateFileInfo(relativePath: string): Object {
```

Comment:
@amitgilad3  please described the correct return type:
Object with 2 props, one is string and one is PathOsBased